go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/spf13/cobra v1.3.0
 	github.com/stretchr/testify v1.7.5
 	github.com/transparency-dev/merkle v0.0.0-20220208131541-728dc2de1344
+	github.com/trustbloc/edge-core v0.1.8
```

Comment:
I don't understand why this package is being added? There shouldn't be any direct dependency on this package anymore.

Reply:
It seems when I tried to import log in one of the new file, I accidently ended up using this package instead of using one of the internal package. I fixed this.